docs/routing/introduction.md

```diff
@@ -0,0 +1,51 @@
+# Introduction
+
+Most JavaScript frameworks come with a dedicated routing solution
+like `angular/router` or `vue-router`. They make it possible to
+navigate through different pages of an application without having to
+do a full page refresh on every link click.
+Since we no longer have one monolithic application that handles all routes, instead we have several independent applications, we need to solve an important issue, namely routing.
```

Comment:
Update:

## Problem statement
When you have a monolithic application, it handles all the routes on its own. When there are two or more applications, they also handle all their routes independently. Since, by default, one independent application know nothing about routes and pages of other independent application. This gives you a problem to solve - a routing issue.